test/csuite/schema_abort/main.c

```diff
@@ -512,14 +512,17 @@ get_all_committed_ts(void)
 
 /*
  * is_set_timestamp_required --
- *     Return if set_timestamp is required.
+ *     Return True if set_timestamp is required.
  */
 static bool
 is_set_timestamp_required(uint64_t last_ts, uint64_t oldest_ts)
 {
-    int64_t ts_diff = (int64_t)(oldest_ts - last_ts);
+    /*
+     * The set_timestamp should be invoked only when current_oldest_ts is greater than
```

Comment:
Instead of `current_oldest_ts` current oldest timestamp and last oldest timestamp in the comment may provide a much more clear picture.

Reply:
Updated the comment here b7c15b29b3ed5e6017f007f54dd792d04a679d8d